amundsen_application/static/js/ducks/dashboard/sagas.ts

```diff
@@ -0,0 +1,20 @@
+import { SagaIterator } from 'redux-saga';
+import { call, put, takeEvery } from 'redux-saga/effects';
+
+import * as API from './api/v0';
+import { getDashboardSuccess, getDashboardFailure } from './reducer';
+import { GetDashboard } from './types';
+
+export function* getDashboardWorker(request): SagaIterator {
```

Comment:
Can we use `action` instead of `request` because that's the current pattern and the object here is indeed Redux action.